python/tvm/relay/frontend/tflite.py

```diff
@@ -2696,11 +2696,11 @@ def convert_transpose_conv(self, op):
 
         # Input (data) Tensor. NHWC layout
         input_tensor = input_tensors[2]
-        _, _, _, input_c = input_tensor.tensor.ShapeAsNumpy()
+        input_shape = input_tensor.tensor.ShapeAsNumpy()
```

Comment:
Don't you think we create `_, input_h, input_w, input_c` is more make sense if you want to use `input_shape[1] / input_shape[2] / input_shape[3]`?

Reply:
Thanks for reviewing.
That seems reasonable. Will do it.